Event/Event.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Gesdinet\JWTRefreshTokenBundle\Event;
+
+use Symfony\Component\EventDispatcher\Event as BaseEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Contracts\EventDispatcher\Event as ContractsBaseEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+if (is_subclass_of(EventDispatcher::class, EventDispatcherInterface::class)) {
```

Comment:
Why not just directly doing this in `Event/RefreshEvent.php` ? There are no other events.

Reply:
I thought this was a bit more elegant solution. The other method involves duplicating the code for the whole RefreshEvent class, AFAIK you can't just put a conditional in the class declaration, it needs to be te whole class code inside the `if` and `else`

Happy to change it though.

Reply:
Not so much code :) but @markitosgv has to decide

Reply:
Thank you both! Nice work @acasademont